src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <section class="card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+        alt="imacIcon"
+        class="card__icon icon"
+      >
+
+      <div class="card__title title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code product-code">
+        Product code: 195434
+      </div>
+
+
+      <div  class="stars stars--star-4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__review review">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-word word">Price:</div>
+        <div class="card__price-digits digits">$2,199</div>
+      </div>
+
+      <div class="card__link">
+        <a
+          class="card__item card__item--hover"
```

Comment:
```suggestion
          class="card__item"
```

remove it, just use :hover in css